examples/tuning/get_important_pattern.py

```diff
@@ -0,0 +1,146 @@
+import itertools
+import pathlib
+from itertools import combinations
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import scikit_posthocs as sp
+import seaborn as sns
+from mlxtend.frequent_patterns import apriori
+from mlxtend.preprocessing import TransactionEncoder
+from networkx import parse_adjlist
+from scipy import stats
+
+metric_name = "acc"
+ascending = False
+
+
+def get_important_pattern(test_accs, vis=True, alpha=0.8, title=""):
```

Comment:
Post hoc pairwise test for multiple comparisons of mean rank sums (Dunn’s test). May be used after Kruskal-Wallis one-way analysis of variance by ranks to do pairwise comparisons